scripts/bulk_update_leads_info.py

```diff
@@ -56,12 +56,15 @@ def get_contact_info(contact_no, csv_row, what, contact_type):
     * opportunity[0-9]_status           - opportunity status
     * opportunity[0-9]_date_won         - opportunity date won
 
-contact columns (new contacts wil be added):
+contact columns (new contacts will be added):
     * contact[0-9]_name                 - contact name
     * contact[0-9]_title                - contact title
     * contact[0-9]_phone[0-9]           - contact phones
     * contact[0-9]_email[0-9]           - contact emails
     * contact[0-9]_url[0-9]             - contact urls
+
+custom fields (new custom types will be created, new fields will be added):
```

Comment:
yeah but this english description doesn't make much sense, also you don't mention create_custom_fields. can you reword it to clarify it? imagine someone using this script that doesn't read the code